src/usage/context/usage.tsx

```diff
@@ -114,22 +115,44 @@ export const UsageProvider: FC<Props> = React.memo(({children}) => {
 
   const handleGetBillingStats = useCallback(async () => {
     try {
-      const resp = await getUsageBillingStats({})
+      // TODO(ariel): making a custom function here because oats can't handle CSV responses?
+      const request = fetch(
+        `${API_BASE_PATH}api/v2/quartz/usage/billing_stats`,
+        {
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+            'Accept-Encoding': 'gzip',
+          },
+        }
+      )
 
-      if (resp.status !== 200) {
-        throw new Error(resp.data.message)
-      }
+      const result = await request.then(processResponse)
 
       // TODO(ariel): keeping this in for testing purposes in staging
       // This will need to be removed for flipping the feature flag on
-      console.warn({resp})
+      console.warn({result})
 
-      const csvs = resp.data?.split('\n\n')
+      if (result.type !== 'SUCCESS') {
+        throw new Error(result.message)
+      }
 
+      const csvs = result?.csv?.split('\n\n')
       // TODO(ariel): keeping this in for testing purposes in staging
       // This will need to be removed for flipping the feature flag on
       console.warn({csvs})
-      setBillingStats(csvs)
+
+      setBillingStats(csvs ?? [''])
+
+      // const resp = await getUsageBillingStats({})
+
+      // if (resp.status !== 200) {
+      //   throw new Error(resp.data.message)
+      // }
+
+      // const csvs = result?.csv?.split('\n\n')
+
+      // setBillingStats(csvs)
```

Comment:
do we need to leave this commented out? Could we remove it and just grab it from the git history in the future?